src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -322,14 +321,18 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
 
         this.loadingProjectAssets = true;
 
-        await this.props.actions.loadAssets(this.props.project);
-        const assets = _.values(this.props.project.assets);
+        const projectAssets = _.values(this.props.project.assets);
```

Comment:
Project file has been updated to only store assets that have been visited/tagged.  This change will merge in project file assets with the assets from the asset/storage provider.